Cargo.toml

```diff
@@ -119,7 +119,7 @@ zip = { version = "0.6", default-features = false, features = ["bzip2"] }
 dice-mfg-msgs = { git = "https://github.com/oxidecomputer/dice-util", default-features = false }
 gateway-messages = { git = "https://github.com/oxidecomputer/management-gateway-service", default-features = false, features = ["smoltcp"] }
 hif = { git = "https://github.com/oxidecomputer/hif", default-features = false }
-humpty = { git = "https://github.com/oxidecomputer/humpty", default-features = false }
+humpty = { git = "https://github.com/oxidecomputer/humpty", default-features = false, branch = "udp-types" }
```

Comment:
I know you're planning to do this already, but commenting as a reminder to switch this back to the main branch after the humpty PR is merged.